ee/codegen/src/generators/nodes/constants.ts

```diff
@@ -26,7 +26,15 @@ export const NODE_ATTRIBUTES: Record<
     },
   },
   ToolCallingNode: {
-    functions: { name: "functions", defaultValue: null },
-    blocks: { name: "blocks", defaultValue: null },
+    blocks: {
+      name: "blocks",
+      defaultValue: null,
+      type: "blocks",
```

Comment:
[nit] I would have these somewhat match the type itself, like `PromptBlocks`. Also if we're going to reuse the value below, let's have the constants assigned to an exported variable